agent_http.go

```diff
@@ -144,7 +144,7 @@ func statusHTTPWriter(w http.ResponseWriter, r *http.Request, deviceManagers []*
 					Dst:             ip.String(),
 					GW:              dm.config.LocalAddress.String(),
 					IsHealthChecked: dm.isHealthChecked(),
-					Healthy:         dm.isHealthy(),
+					Healthy:         dm.healthCheck.healthy,
```

Comment:
ha, an instinct prevents me from using unexported fields as when main becomes bigger it makes me uneasy and I keep thinking that I will split stuff into smaller packages. But that is ofc fine as well

Reply:
Cool - when we do, we'll create a function. Right now it makes no sense )